### PR TITLE
Add lua control_structures

### PR DESCRIPTION
**Pull request by cafce25**
## What changed and why?

   Add lua as a language and specifically its control_structures.

## Checklist

   

- [x] I claimed any associated issue(s) and they are not someone else's
- [x] I have looked at documentation to ensure I made any revisions correctly
- [x] I tested my changes locally to ensure they work

### CODE REVIEW

File: web/thesauruses/lua/5/control_structures.json [hunk 1]
```diff
@@ -0,0 +1,120 @@
+{
+  "meta": {
+    "language": "lua",
+    "language_name": "Human-Readable Language Name",
```
Comment (geekygirlsarah) [inline web/thesauruses/lua/5/control_structures.json]: You meant to update this from "Human-Readable Language Name", right? :)

**Review by geekygirlsarah — CHANGES_REQUESTED** [review]
Update the language_name and I'll merge it!

(Yay, a new language!)

File: web/thesauruses/lua/5/control_structures.json (hunk 1 above)
Comment (cafce25) [reply web/thesauruses/lua/5/control_structures.json]: Huh? I'm pretty sure i edited this, ... I wonder which git command I used to undo this.

**Review by geekygirlsarah — APPROVED** [review]
Sweet, thanks for fixing it. I'll merge it in!

(I think I'm more curious about why the file checker didn't detect this...)

**Comment by geekygirlsarah** [comment]
Now I get to make issues for the Lua holes now...

**Pull request merged**